.ruby-version

```diff
@@ -0,0 +1 @@
+ruby-2.1.1
```

Comment:
I'm pretty sure this codebase is still running on 1.9.3-p545 in production so that is probably want we want to use here.

Notably this codebase runs in two places, bundled in as part of [the api](https://github.com/librato/api) for definitions and on the gobot nodes for actual notifications. I know the API is still on 1.9.3, @mheffner or @collinvandyck do you know about the gobot nodes?